pkg/api/schemav2/schemav2.proto

```diff
@@ -69,6 +69,31 @@ message DocumentSearchResponse {
   uint32 entriesLeft = 4;
 }
 
+message DocumentProofRequest {
+  string collection = 1;
+  map<string, PossibleIndexType> primaryKeys = 2;
+  int64 atRevision = 3;
+}
+
+message DocumentAuditRequest {
+  string collection = 1;
+  map<string, PossibleIndexType> primaryKeys = 2;
```

Comment:
not sure I understand the objective of `primaryKeys` here based on the defined type